### PR TITLE
Update Cloud Quickstart

### PR DESCRIPTION
**Pull request by mikeCRL**

### CODE REVIEW

File: src/current/cockroachcloud/quickstart.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-summary: Learn how to create and use a CockroachDB Standard cluster.
+summary: Learn how to use free trial credits to deploy a CockroachDB cluster on CockroachDB Standard (Preview), then use sample code to run your first queries.
 toc: true
 cloud: true
 referral_id: docs_quickstart_free
 docs_area: get_started
 ---
 
-This page shows you how to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (free for a 30-day trial for your first cluster), connect to it using a sample workload, and run your first query.
+This page shows you how to use free trial credit to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview), and use sample code to run your first queries.
```
Comment (jaiayu) [inline src/current/cockroachcloud/quickstart.md:10]: "free trial credit" should be plural

**Review by jaiayu — APPROVED** [review]

File: src/current/cockroachcloud/quickstart.md (hunk 1 above)
Comment (mikeCRL) [reply src/current/cockroachcloud/quickstart.md:10]: SGTM. Thanks!

File: src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md [hunk 2]
```diff
@@ -1,18 +1,14 @@
-{% comment %}
-After the free trial details are finalized, add a link to free trial page to check for free trial eligibility.
-{% endcomment %}
-
-1. Create a [CockroachDB {{ site.data.products.cloud }}](https://cockroachlabs.cloud/signup?referralId={{page.referral_id}}) account and log in.
-1. On the **Clusters** page, click **Create cluster**.
+1. [Create a CockroachDB {{ site.data.products.cloud }} account]({% link cockroachcloud/create-an-account.md %}). If this is your first CockroachDB Cloud organization, it will be credited with $400 in [free trial credits]({% link cockroachcloud/free-trial.md %}) to get you started.
```
Comment (mdlinville) [inline src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md:1]: Noting that this content must somehow be kept up to date if the actual amount of credits changes.

File: src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md [hunk 3]
```diff
@@ -1,18 +1,14 @@
...
+1. On the **Get Started** page, click **Create cluster**.
 1. On the **Select a plan** page, select **Standard**.
-1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS) in the **Cloud provider** section.
+1. On the **Cloud & Regions** page, select a cloud provider (GCP or AWS).
```
Comment (mdlinville) [inline src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md:4]: If we avoid the parenthesis we don't need to update this when you can choose Azure.

File: src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md [hunk 4]
```diff
@@ -1,18 +1,14 @@
...
 1. In the **Regions** section, select a region for the cluster. Refer to [CockroachDB {{ site.data.products.cloud }} Regions]({% link cockroachcloud/regions.md %}) for the regions where CockroachDB {{ site.data.products.standard }} clusters can be deployed. To create a multi-region cluster, click **Add region** and select additional regions.
 1. Click **Next: Capacity**.
-1. On the **Capacity** page, keep the **Provisioned capacity** at the default value of 1000 RUs/second.
+1. On the **Capacity** page, keep the [**Provisioned capacity**]({% link cockroachcloud/plan-your-cluster.md %}) at the default value of 2 vCPUs.
 
     Click **Next: Finalize**.
 
-1. On the **Finalize** page, name your cluster. If applicable, the 30-day trial code is pre-applied to your cluster.
+1. On the **Finalize** page, name your cluster. If an active free trial is listed in the right pane, you will not need to add a payment method, though you will need to do this [by the end of the trial]({% link cockroachcloud/free-trial.md %}#add-payment-methods) to further maintain your organization and its clusters.
```
Comment (mdlinville) [inline src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md:11]:
```suggestion
1. On the **Finalize** page, optionally name your cluster or choose the randomly-assigned name. If an active free trial is listed in the right pane, you will not need to add a payment method. Before the  [end of the trial]({% link cockroachcloud/free-trial.md %}#add-payment-methods), you must add a payment method to keep using the cluster..
```
I do not believe you need a payment method to keep the organization. Is it worth noting that you will be reminded about this near the end of the trial?

File: src/current/cockroachcloud/create-an-account.md [hunk 5]
```diff
@@ -39,17 +38,13 @@ We highly recommend enabling multi-factor authentication (MFA) with your SSO pro
...
-1. Enter your **Email address**.
-1. Enter your **Name**.
-1. Enter a **Password**.
-1. Enter your **Organization name (optional)**.
-
-    This can be [changed](#change-your-organization-name) later.
-
-1. Select the checkbox to accept the [terms of service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [privacy policy](https://www.cockroachlabs.com/privacy).
+1. On the [registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account), click **Sign up with email**.
+1. Enter your **Work email** and a unique **Password** for CockroachDB Cloud.
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-an-account.md:42]: Is there a reason we need to be this prescriptive?
```suggestion
1. Enter an email address and password for CockroachDB Cloud.
```

File: src/current/cockroachcloud/create-an-account.md [hunk 6]
```diff
@@ -39,17 +38,13 @@ We highly recommend enabling multi-factor authentication (MFA) with your SSO pro
...
+1. Select the checkbox to accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy).
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-an-account.md:43]:
```suggestion
1. Accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy), then click **Get Started**.
```
If you take this suggestion, you can eliminate the next step.

File: src/current/cockroachcloud/quickstart.md [hunk 7]
```diff
@@ -1,13 +1,13 @@
 ---
 title: Quickstart with CockroachDB
-summary: Learn how to create and use a CockroachDB Standard cluster.
+summary: Learn how to use free trial credits to deploy a CockroachDB cluster on CockroachDB Standard (Preview), then use sample code to run your first queries.
```
Comment (mdlinville) [inline src/current/cockroachcloud/quickstart.md:3]:
```suggestion
summary: Learn how to use free trial credits to deploy a CockroachDB cluster on CockroachDB Standard (Preview) and run your first queries.
```

File: src/current/cockroachcloud/quickstart.md [hunk 8]
```diff
@@ -1,13 +1,13 @@
...
 toc: true
 cloud: true
 referral_id: docs_quickstart_free
 docs_area: get_started
 ---
 
-This page shows you how to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (free for a 30-day trial for your first cluster), connect to it using a sample workload, and run your first query.
+This page shows you how to use free trial credits to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview), and use sample code to run your first queries.
```
Comment (mdlinville) [inline src/current/cockroachcloud/quickstart.md:10]:
```suggestion
This page shows you how to use free trial credits to deploy a CockroachDB cluster on CockroachDB {{ site.data.products.standard }} (Preview) and use sample code to run your first queries.
```

**Review by mdlinville — APPROVED** [review]

File: src/current/cockroachcloud/create-an-account.md (hunk 5 above)
Comment (mikeCRL) [reply src/current/cockroachcloud/create-an-account.md:42]: Confused, because I thought I made the same edit! Thanks.

**Review by mikeCRL — COMMENTED** [review]
TFTR @mdlinville . Implementing a version of most changes and merging.

File: src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md (hunk 2 above)
Comment (mikeCRL) [reply src/current/_includes/cockroachcloud/quickstart/create-free-trial-standard-cluster.md:1]: Ack. We will have to search for other mentions anyway.

File: src/current/cockroachcloud/create-an-account.md [hunk 9]
```diff
@@ -39,8 +39,8 @@ We highly recommend enabling multi-factor authentication (MFA) with your SSO pro
 
 <section class="filter-content" markdown="1" data-scope="email">
 1. On the [registration page](https://cockroachlabs.cloud/signup?referralId=docs_create_account), click **Sign up with email**.
-1. Enter your **Work email** and a unique **Password** for CockroachDB Cloud.
-1. Select the checkbox to accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy).
+1. Enter an email address and password for CockroachDB Cloud.
+1. Accept the [Terms of Service](https://www.cockroachlabs.com/cloud-terms-and-conditions/) and [Privacy Policy](https://www.cockroachlabs.com/privacy), then click **Get Started**.
 1. Click **Get Started**.
```
Comment (mdlinville) [inline src/current/cockroachcloud/create-an-account.md:44]:
```suggestion
```

**Review by mdlinville — APPROVED** [review]

**Pull request merged**